Emitron/Emitron/Models/PlaybackSpeed.swift

```diff
@@ -66,3 +66,10 @@ enum PlaybackSpeed: Int, CaseIterable, SettingsSelectable {
     allCases
   }
 }
+
+extension PlaybackSpeed {
+  // Added hash(into: ) funciton because of this bug: https://bugs.swift.org/browse/SR-13851
+  func hash(into hasher: inout Hasher) {
+    hasher.combine(rate)
```

Comment:
Any particular reason you're using `rate` here for the hash instead of `display` like the rest of the settings?

Reply:
@0xTim no particular reason, will change it to `display` to match the rest of the settings